app/src/main/java/org/wikipedia/bridge/JavaScriptActionHandler.kt

```diff
@@ -38,20 +39,20 @@ object JavaScriptActionHandler {
     }
 
     @JvmStatic
-    fun setUp(topMargin: Int): String {
+    fun setUp(): String {
         val app: WikipediaApp = WikipediaApp.getInstance()
-        return String.format("pcs.c1.Page.setup({" +
-                "platform: pcs.c1.Platforms.ANDROID," +
-                "clientVersion: '%s'," +
-                "theme: pcs.c1.Themes.%s," +
-                "dimImages: %b," +
-                "margins: { top: '%dpx', right: '%dpx', bottom: '%dpx', left: '%dpx' }," +
-                "areTablesInitiallyExpanded: %b," +
-                "textSizeAdjustmentPercentage: '100%%'," +
-                "loadImages: %b" +
-                "})", BuildConfig.VERSION_NAME, app.currentTheme.funnelName.toUpperCase(),
+        return String.format("{" +
+                "\"platform\": \"pcs.c1.Platforms.ANDROID\"," +
+                "\"clientVersion\": \"%s\"," +
+                "\"theme\": \"%s\"," +
+                "\"dimImages\": \"%b\"," +
+                "\"margins\": { \"top\": \"%dpx\", \"right\": \"%dpx\", \"bottom\": \"%dpx\", \"left\": \"%dpx\" }," +
+                "\"areTablesInitiallyExpanded\": \"%b\"," +
+                "\"textSizeAdjustmentPercentage\": \"100%%\"," +
+                "\"loadImages\": \"%b\"" +
+                "}", BuildConfig.VERSION_NAME, app.currentTheme.funnelName,
```

Comment:
Not sure why to change the string with escaped slashes. Can we use the previous format since it looks better?

Reply:
the pcsClient expects the string that is to be returned by getSetupSettings() in that format @cooltey . Earlier it was like theme:dark, now it needs to be like "theme":"dark". Hence the escaping.

Reply:
I see. Is it possible to use a single quote to them?

Reply:
tried... but no.. only in double quotes " "